installer/templates/phx_new/lib/app_name/web.ex

```diff
@@ -0,0 +1,76 @@
+defmodule <%= application_module %>.Web do
+  @moduledoc """
+  A module that keeps using definitions for controllers,
+  views and so on.
```

Comment:
The blog module would wrap up all access so we wouldn't import Repo aliases or functions. Instead there will be generated functions like `Blog.list_comments`, `Blog.get_comment(id)`, that internally hit the repo. As the user needs to extend their generated code, they would put custom queries and options within the context, with the list/get ones as a guide. The caller will remain unaware of repo access or Ecto query DSL.